### ISSUE TITLE
Add offset and limit based on batch size for Aggregate SOQL Queries to make them batchable

### ISSUE THREAD

**Issue by datapharmer** [douglascayers-org/sfdx-mass-action-scheduler#75]
Querymore() isn't supported for aggregate queries, but by using a sort field that is unique (such as an ID), and using a combination of limit and offset based on batchsize it is possible to iterate through all available results matching the query (up to the soql query governors of course!) This would help where the total number of results is unknown, but it still far more efficient to aggregate that no launch new processes and then try to handle duplicates and conflicts as part of the workflows that are launched.

**Comment by douglascayers** [comment]
Thanks for the feedback @datapharmer. Do you happen to have any sample code for using this strategy in batch apex?

**Comment by douglascayers** [comment]
Hi @datapharmer, I believe this has been resolved in a prior release, or, if this was in reference to when a query was being validated on the Mass Action Configuration page and you got this error, then that is being resolved in v2.3.

In the upcoming release, this aggregate query returns and processes 1,000+ accounts:
```
SELECT Id, COUNT(Name) FROM Account GROUP BY Id ORDER BY Id
```

**Comment by douglascayers** [comment]
Release 2.3 is now available per #99, please see the [release notes](https://github.com/douglascayers-org/sfdx-mass-action-scheduler/wiki/Release-Notes-v2.3) for full details. Thank you!
